### ISSUE TITLE
About "x_0" and "x_next" in the diffusion.py

### ISSUE THREAD

**Issue by shrxht** [Kroery/DiffMOT#12]
```
                x0 = self.pred_x0_from_xt(x_t, noise_pred, C_pred, cur_time)
                x0.clamp_(-1., 1.)
                C_pred = -1 * x0
                x_next = self.pred_xtms_from_xt(x_t, noise_pred, C_pred, cur_time, s)
```

why `traj[t-1] = x_next.detach()`, why not `traj[t-1] = x_0.detach()`

**Comment by GuHuangAI** [comment]
@shrxht 
You can seem x0 as an estimation in each denoising step, and x_next is the real denoised signal.
